src/django/oar/urls.py

```diff
@@ -29,6 +29,10 @@
     url(r'^api-auth/', include('rest_framework.urls')),
     url(r'^rest-auth/', include('rest_auth.urls')),
     url(r'^rest-auth/registration/', include('rest_auth.registration.urls')),
+    url(r'^user-login/', views.LoginToOARClient.as_view(),
```

Comment:
Looking at the docs, django-rest-auth appears to have login and logout routes baked in https://django-rest-auth.readthedocs.io/en/latest/api_endpoints.html

If we need to override them for some reason, I think we should update the commit message to explain why.

Reply:
Will do!

Reply:
The reason I added the override was to enable making a GET request for session sign in. The base view supports POST for login out of the box, but session signin is a GET.

Since the commit itself is currently buried behind a bunch of fixups, I'll wait to edit its message until I'm rebasing and know in what commit this changes ends up living.